### DISCUSSION TITLE
How to use PC mods?

### DISCUSSION THREAD

**Discussion by Lawlessly** [thatskymod/Sky-CotL-Scripts#129] [Q&A] [answered]
Hi I'm pretty new to this stuff. I've been trying to use the PC mods but nothing seems to work. I followed the steps and I believe I extracted everything into the right places, but nothing happens when I open the game. Any help would be appreciated

![Screenshot 2024-09-03 225922](https://github.com/user-attachments/assets/2e665505-a87f-4746-9b81-85568f76c2d1)
![Screenshot 2024-09-03 230052](https://github.com/user-attachments/assets/ad6c3022-2e36-4e6a-baad-7163f92decb9)

**Comment by thatskymod** [comment, marked as answer]
I see that you've almost got it right, but there's a small adjustment needed. You should extract the contents of `sml-pc.zip` directly into the directory where Sky.exe is located, ensuring that the files are not nested within an additional subfolder.

The correct directory structure looks like this:

```
C:\Program Files (x86)\Steam\steamapps\common\Sky Children of the Light
├───mods/                      // this is where your mods reside
│   ├───demo.dll
│   ├───TSM.dll
│   └───TSM Resources/
│       ├───Languages/
│       ├───Music Sheets/
│       ├───Outfits/
│       ├───Teleport Locations/
│       └───TSM.json
├───crashpad_handler.exe       // this is your Sky folder
├───fmod.dll
├───fmodstudio.dll
├───libcurl.dll               // and libcurl.dll from SML
├───powrprof.dll              // and so on...
├───Sky.exe
├───Sky.lib
├───Sky.log
├───Sky.res
├───sml_config.json
└───steam_api64.dll
```

**Reply by Lawlessly** [reply]
Thank you 🙏
